yaml2rst.py

```diff
@@ -101,6 +101,9 @@ def convert(lines):
         elif line.startswith('# ') or line == '#':
             if state != STATE_TEXT:
                 yield ''
+            # Filter out [[[\d and ]]] at the end of a documentation line.
+            # Those sequences can be used for folding sections.
+            line = re.sub(r"\s*(:?\[{3}|\]{3})\d?$", "", line)
```

Comment:
- This would strip foldmarks in every line, not only in rst-comments
- `\s*` is of no use?
- What should the trailing digit be?